Add notes for Learn to Use Vim lesson

FabioRosado opened the pull request:
Hello everyone,

this is my first contribution to the notes repo. I've added notes for the [Learn to use VIM](https://egghead.io/courses/learn-to-use-vim) lesson.

Will look forward to hearing your feedback 😁 👍

lsminter commented on learn-to-use-vim/06-combining-vim-commands.md:
```diff
@@ -0,0 +1,18 @@
+
+## Key Combo Trick
+
+- Delete text in between <div> tags by using the keys `C`-`I`-`T`.
```
Here I think you need backticks around the `<div>` or else it creates a new line.

lsminter commented on learn-to-use-vim/10-introduction-to-vim-plugins.md:
```diff
@@ -0,0 +1,12 @@
+# Introduction to Vim Plugins
```
I think it would be really nice to have an example of how to use Plugins Managers or just how to add a plugin using a manager would be very beneficial.

lsminter commented on learn-to-use-vim/README.md:
```diff
@@ -0,0 +1,46 @@
+<h1 align="center"><a href="https://egghead.io/courses/learn-to-use-vim">Learn to use Vim</a></h1>
+<p align="center"><img src="https://d2eip9sf3oo6c2.cloudfront.net/series/square_covers/000/000/045/full/EGH_Vim.png" width="200></p>
```
Took me a while to figure out why the About section wasn't showing up. You forgot the second " after the width in the img.

FabioRosado replied:
You know what, I look at the preview and noticed that something looked funny, but I thought it was some extension that I installed on my vscode since it was complaining that I can't have HTML code in markdown haha

Thank you for spotting this!

lsminter commented:
Oh also put the notes a notes folder.

FabioRosado commented:
Just updated this PR to address all your topics Lucas. I've also fixed some issues that I have noticed when I was writing the cheatsheet.

I used adobe xd to design the cheatsheet and added the file to the PR, not sure if this is alright 🤔  I think next time I should just use figma and add the link there so other folks could update the cheatsheet if needed 😁

lsminter commented:
I love the cheatsheet! Adding a PDF works perfectly. I'm unsure if there is much that could be added to the cheatsheet if anything. 

I'm having trouble finding the notes now though. All I see is the cheatsheet. I see that you were moving them into a notes folder but I don't see the notes folder in the course folder.

FabioRosado commented:
Uff you know what? I use a `NOTES` file to keep track of my notes and had that added to my global gitignore. I didn't notice but moving the notes to the folder, it just got ignored by git haha

Fixed the issue, thanks for pointing that out! 😁

lsminter commented:
Dope! Sorry for the late response. I'll get this merged and added to egghead!
